demoapp/DEVELOPMENT.md

```diff
@@ -0,0 +1,6 @@
+**REQUIRES a MacOS Host to build all modules**
+
+* recursively clone this repo
+* set `sdk.dir=/absulute/path/to/Android/sdk` inside `kmp-crypto/local.properties`
+* `cd kmp-crypto && ./gradlew publishAllPublicationsToLocalRepository`
```

Comment:
@JesusMcCloud i don't think this is accurate anymore? i don't remember having to recursively clone anything, or needing to run gradle commands :9

Reply:
swift-klib is still pulled-in recursively and the swift code is used by the verifier. but publishing is not needed anymore